.github/set-versions.sh

```diff
@@ -37,6 +37,8 @@ if [ -z "${JAVAONLY}" ] ; then
         sed -i"${VERIFY}" -e 's/^version = ".*"$/version = "'"${VERS}"'"/' "${TOMLFILE}"
         EDITEDFILES="${EDITEDFILES} ${TOMLFILE}"
     done
+
+    sed -i"${VERIFY}" -e 's/^Version: .*$/Version = '"${VERS}"'/' cpp/ironoxide.pc.in
```

Comment:
I assume this just replaces the version line in the `.pc.in` file, but should we change it below so that it has something like `Version: {VERSION}` instead of having it hardcoded to some random value that isn't correct? Then it will maybe be more obvious that it's a dynamic value that gets filled in at release time.

Reply:
It's kept up to date by this process; it's never set to a random, incorrect value. Same story as all the other version files it has to maintain; they're all kept in sync with each other. The process is briefly documented in `RELEASING.md`. Do you think we need a comment in this file pointing to that doc?